app/sqaws.py

```diff
@@ -165,8 +268,22 @@ def estimate_monthly_ebs_storage_price(region_name: str, instance_id: str) -> fl
     return total_gb * 0.1  # Assume EBS costs $0.1/GB/month, true as of Dec 2021 for gp2 type storage
 
 
-# Set a tag on an instance
-def set_tag(region_name: str, instance_id: str, tag_name: str, tag_value: str, dryrun: bool) -> None:
+def estimate_monthly_volume_price(volume_type: str, size: float, iops: float, throughput: float) -> float:
```

Comment:
A reference to the basic calculation will be helpful. Something like the comment above.
`# Assume EBS costs $0.1/GB/month, true as of Dec 2021 for gp2 type storage`

Reply:
Awesome, just added that